core/src/main/java/com/microsoft/applicationinsights/internal/config/ApplicationInsightsXmlConfiguration.java

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Maybe it worth keeping the naming convention as other class and call it ApplicationInsightsXmlElement.java